### ISSUE TITLE
Proposal: Context Menu

### ISSUE THREAD

**Issue by mlucool** [jupyterlab/jupyterlab-toc#138]
I'd like to propose we add a context menu for all levels of the TOC. While I am sure there would be other use cases, I'd think a good and first use case is "Run cells". What do you think?

**Comment by kgryte** [comment]
@mlucool That seems reasonable; however, there is currently some debate regarding whether the ToC extension will be merged into JLab core, so not clear whether feature development is effectively frozen until its fate is resolved.

**Comment by mlucool** [comment]
I would be a fan of this being a core feature. Is there a timeline on determining this?

**Comment by kgryte** [comment]
Not sure the timeline. Partly because core devs are evaluating whether the act of merging into core should precipitate a reimplementation for purposes of tighter integration, etc. I can raise this question at the next dev meeting.

**Comment by kgryte** [comment]
@mlucool See https://github.com/jupyterlab/jupyterlab-toc/pull/141. Let me know if this works for your use case.

**Comment by mlucool** [comment]
That looks great! One thing that I assume was only for POC reasons was the need to "toggle code cells" first. It should always be available IMO.

**Comment by kgryte** [comment]
@mlucool Agreed on being more generally available; however, sadly, this would require a more extensive refactoring of the extension (and time commitment) to support this.
